fenicsxconcrete/finite_element_problem/linear_elasticity.py

```diff
@@ -1,16 +1,23 @@
 import dolfinx as df
 import numpy as np
 import ufl
+import pint
 from petsc4py.PETSc import ScalarType
 from fenicsxconcrete.helper import Parameters
 from fenicsxconcrete.unit_registry import ureg
 from fenicsxconcrete.finite_element_problem.base_material import MaterialProblem
+from fenicsxconcrete.experimental_setup.cantilever_beam import CantileverBeam
+from fenicsxconcrete.experimental_setup.base_experiment import Experiment
 
 
 class LinearElasticity(MaterialProblem):
     """Material definition for linear elasticity"""
 
-    def __init__(self, experiment, parameters, pv_name='pv_output_linear_elasticity', pv_path=None):
+    def __init__(self,
```

Comment:
See above

Reply:
I think I forgot to sumbmit one comment: Please add the following header to the init

```python
    def __init__(self,
                 experiment: Experiment,
                 parameters: dict[str, pint.Quantity],
                 pv_name : str='pv_output_full',
                 pv_path : str=None):
```
You only had the type hint for the parameters;)

Reply:
done